File: main/java/com/ruthlessjailer/api/poseidon/example/Poseidon.java
package com.ruthlessjailer.api.poseidon.example;

import com.ruthlessjailer.api.poseidon.Chat;
import com.ruthlessjailer.api.poseidon.PluginBase;

/**
 * @author RuthlessJailer
 */
public class Poseidon extends PluginBase {

	@Override
	public void onStart() {
		Chat.INSTANCE.setDebugMode(true);
		registerCommands(new TestCommand());
	}

}

File: src/main/java/com/ruthlessjailer/api/poseidon/example/Poseidon.kt
package com.ruthlessjailer.api.poseidon.example

import com.ruthlessjailer.api.poseidon.Chat
import com.ruthlessjailer.api.poseidon.PluginBase

/**
 * @author RuthlessJailer
 */
class Poseidon : PluginBase() {

	override fun onStart() {
		Chat.debugMode = true

		registerCommands(TestCommand)
		println("YEETED")
	}

}